client/ayon_core/hosts/nuke/api/push_to_project.py

```diff
@@ -0,0 +1,116 @@
+from collections import defaultdict
+import shutil
+import os
+
+from ayon_api import get_project, get_folder_by_id, get_task_by_id
+from ayon_core.settings import get_ayon_settings, get_project_settings
+from ayon_core.pipeline import Anatomy, registered_host
+from ayon_core.pipeline.template_data import get_template_data
+from ayon_core.pipeline.workfile import get_workdir_with_workdir_data
+from ayon_core.tools.push_to_project.main import main_show
+
+from .utils import bake_gizmos_recursively
+
+import nuke
+
+
+def bake_container(container):
+    """Bake containers to read nodes."""
+
+    node = container["node"]
+
+    # Fetch knobs to remove in order.
+    knobs_to_remove = []
+    remove = False
+    for count in range(0, node.numKnobs()):
+        knob = node.knob(count)
+
+        # All knobs from "OpenPype" tab knob onwards.
+        if knob.name() == "AYON":
```

Comment:
This change requires to add import `from .lib import MENU_LABEL`
```suggestion
        if knob.name() == MENU_LABEL:
```